tests/test_routing.py

```diff
@@ -224,3 +224,23 @@ def get_path(var: int, path: http.Path):
         response = client.get('/%d/' % index)
         assert response.status_code == 200
         assert response.json() == {'path': '/%d/' % index}
+
+
+def test_routing_reversal_on_path_without_url_params():
+    url = app.router.reverse_url('found')
+    assert url == 'http://localhost:8080/found/'
+
+
+def test_routing_reversal_on_path_non_existent_path():
+    with pytest.raises(exceptions.NotFound):
```

Comment:
We'll have something like a `NoReverseMatch` (resulting in a 500 error), rather than `NotFound` (which would imply a 404 response)